charts/jenkins/CHANGELOG.md

```diff
@@ -12,13 +12,18 @@ Use the following links to reference issues, PRs, and commits prior to v2.6.0.
 The changelog until v1.5.7 was auto-generated based on git commits.
 Those entries include a reference to the git commit to be able to get more details.
 
+## 4.9.0
```

Comment:
I assume the intention is the comment about changelog processing goes above the changelog entry?

cc @lemeurherve

Reply:
Yes